azure-pipelines.yml

```diff
@@ -290,6 +290,35 @@ jobs:
       ci/run_verilator_pytest.sh
     displayName: 'Execute tests'
 
+- job: "riscv_compliance_tests"
+  displayName: "Execute RISC-V compliance tests"
+  pool: "Default"
+  dependsOn:
+    - top_earlgrey_verilator
```

Comment:
@mcy I am not sure if this is dependency is 100% necessary.
When I did not have it earlier, I think one of the jobs actually failed, and I just assumed it was a verilator conflict over the pty.

I'm actually not sure how isolated these jobs are supposed to be..

Reply:
This dependency is necessary, as you need to build the simulation first.

Reply:
sorry i was actually referring to the execute* dependency, not the build dependency.